addons/uom/static/src/js/uom_field_widget.js

```diff
@@ -0,0 +1,147 @@
+odoo.define('uom.UomFieldWidget', function (require) {
+"use strict";
```

Comment:
by convention, we now indent the first level as well (the whole module content should be indented)